docker/README.md

```diff
@@ -0,0 +1,149 @@
+# Building KubeStellar deployment as a container
+
+Table of contents:
+- [KubeStellar in a container](#kubestellar-in-a-container)
+  - [Install the pre-requisites](#install-the-pre-requisites)
+  - [Build a local container image](#build-a-local-container-image)
+  - [Build and push a multi-architecture image](#build-and-push-a-multi-architecture-image)
+  - [Run the container and access KubeStellar from the host OS](#run-the-container-and-access-kubestellar-from-the-host-os)
+  - [Checking the container logs](#checking-the-container-logs)
+  - [Login into the running container to use **KubeStellar** from inside the container](#login-into-the-running-container-to-use-kubestellar-from-inside-the-container)
+  - [Teardown and cleanup](#teardown-and-cleanup)
+
+## Install the pre-requisites

```

Comment:
The instructions here look specific to the Debian family of Linux distros.  This specificity should be stated up front.

It would be good to separate building and pushing the image from using the image.

Reply:
I worked a bit on the text

Reply:
I see that "Several examples are shown below. On Debian systems, execute this command:" is included on line 17.  I think this satisfies the requirement.